service/history/api/signalwithstartworkflow/signal_with_start_workflow.go

```diff
@@ -80,7 +80,7 @@ func SignalWithStartWorkflow(
 
 func startAndSignalWorkflow(
 	ctx context.Context,
-	shard shard.Context,
+	shardCtx shard.Context,
```

Comment:
Had to rename this since it clashed with the package name and gave a compilation error.